k8s/examples/postgres/pgbench/25-postgres-pgbench.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pgbench
+spec:
+  containers:
+    - name: pgbench
+      image: crunchydata/crunchy-pgbench:centos7-10.8-2.4.0-rc9
```

Comment:
I couldn't pull this image

```
  Warning  Failed            18s (x3 over 1m)  kubelet, ip-192-168-56-100.eu-west-1.compute.internal  Failed to pull image "crunchydata/crunchy-pgbench:centos7-10.8-2.4.0-rc9": rpc error: code = Unknown desc = Error response from daemon: manifest for crunchydata/crunchy-pgbench:centos7-10.8-2.4.0-rc9 not found
  Warning  Failed            18s (x3 over 1m)  kubelet, ip-192-168-56-100.eu-west-1.compute.internal  Error: ErrImagePull
  Normal   BackOff           5s (x3 over 1m)   kubelet, ip-192-168-56-100.eu-west-1.compute.internal  Back-off pulling image "crunchydata/crunchy-pgbench:centos7-10.8-2.4.0-rc9"
  Warning  Failed            5s (x3 over 1m)   kubelet, ip-192-168-56-100.eu-west-1.compute.internal  Error: ImagePullBackOff
```

Reply:
The postgres images were also RC images. I've changed them to what look like stable images.